deepspeed/ops/fp_quantizer/quantize.py

```diff
@@ -42,12 +42,18 @@ def dequantize(self, input_q, fp_out=None, q_bits=8, q_mantisa_bits=3, scale=Non
 
 class FP_Quantize(Quantizer):
 
-    def __init__(self, group_size=512) -> None:
+    def __init__(self, quantization_config) -> None:
         global fp_quant_module
-        super().__init__(group_size=group_size)
+        super().__init__(group_size=quantization_config.group_size)
         if fp_quant_module is None:
             fp_quant_module = FPQuantizerBuilder().load()
+        self.is_python_impl = getattr(fp_quant_module, "PYTHON_IMPL", False)
```

Comment:
It seems the goal here is to handle `cuda` specially, so I think it is make that explicit with something like:

```suggestion
        self.is_cuda_impl = getattr(fp_quant_module, "CUDA_IMPL", False)
```

Reply:
@tjruwase you are right. It is better to do it that way.